pkgs/build-support/fetch-node-modules.nix

```diff
@@ -25,6 +25,9 @@ stdenvNoCC.mkDerivation ({
 
   phases = "unpackPhase patchPhase buildPhase installPhase";
 
+  # npm doesn't support var `SSL_CERT_FILE`.
+  NODE_EXTRA_CA_CERTS = "${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
How did you find out about this CA issue? https://github.com/NixOS/nixpkgs/pull/257513 suggests that npm has hardcoded CAs.

Reply:
Without `NODE_EXTRA_CA_CERTS`, the `npm ci` command in `fetchNodeModules` hangs.
You can repro this by setting the hash in `backend = fetchNodeModules` to an empty string and then running `nix build --no-link -L .#mempool-backend`.

Reply:
Hm ok, good that we discovered this!